cmd/list_test.go

```diff
@@ -62,6 +62,18 @@ func TestListWithIncorrectLimit(t *testing.T) {
 	assert.True(t, gock.IsDone(), internal.PrintPendingMocks(gock.Pending()))
 }
 
+func TestListWithIncorrectLimit2(t *testing.T) {
```

Comment:
Thanks for your PR, it looks almost good to me, just one small comment - 

Would you want to rename the test to something more meaningful as there's no difference in what this test and `TestListWithIncorrectLimit` are doing name and test wise. Feel free to call out something like, `TestListLimitShorthandUsingIncorrectLimit` that would tell the intention behind adding a similar test.

Reply:
Thank you so much for your review. I renamed the function as you said.